.meteor/packages

```diff
@@ -170,3 +170,5 @@ yasaricli:slugify
 yasinuslu:blaze-meta
 deepwell:bootstrap-datepicker2
 rocketchat:google-natural-language
+rocketchat:drupal
+rocketchat:i18n
```

Comment:
not sure. I just used the build example and added rocketchat:drupal using the `meteor add rocketchat:drupal` command. that I take it added the lines to the package. I am new to meteor, but expierenced in drupal so do not be afraid to tell me i did something wrong ;) I prefer to do it right than muddle on in a 'wrong' way.